extra/statistics-generator

```diff
@@ -78,6 +78,9 @@ collect_changes() {
             pushd .
         else
             CHANGES="$("$RELEASE_TOOL" --version-of $repo --in-integration-version "$given_range")"
+            if [[ "${CHANGES}" = "" ]]; then
+                continue
+            fi
```

Comment:
How can this happen? This seems more like a user error or bug, which I'd want to know about.

Reply:
That is also my suspicion :wink:

Reply:
Like @lluiscampos figured out, it was because I was using master for `2.4`